src/spin_sdk/sqlite.py

```diff
@@ -1,33 +1,28 @@
 """Module for interacting with an SQLite database"""
 
 from typing import List
-from spin_sdk.wit.imports.sqlite import Connection, ValueInteger, ValueReal, ValueText, ValueBlob
```

Comment:
The sqlite example needs to be updated as it relies on the `Value_Integer`

Reply:
Good catch; fixed.